### ISSUE TITLE
1.14.0 Exception

### ISSUE THREAD

**Issue by pson** [jrossignol/ContractConfigurator#542]
Good afternoon (well it was for me when I wrote this), I got an exception and was told to post this here. If I can do anything to help locating the bug, just ask. Thank you for your work, and have a nice summer (unless you are in the southern hemisphere.)

Best wishes, P:son
```
Exception occured while loading contract 'CleverSat.CleverSatProbeMalfunction':
System.Exception: Error parsing statement.
Error occurred near '*':

* <-- HERE ---> System.NullReferenceException: Object reference not set to an instance of an object
  at ContractConfigurator.ExpressionParser.ExpressionParser`1[ContractConfigurator.VesselIdentifier].ParseSimpleStatement[VesselIdentifier] () [0x00000] in <filename unknown>:0 
  --- End of inner exception stack trace ---
  at ContractConfigurator.ExpressionParser.ExpressionParser`1[T].ParseExpression (System.String key, System.String expression, ContractConfigurator.ExpressionParser.DataNode dataNode) [0x00000] in <filename unknown>:0 
  at ContractConfigurator.ExpressionParser.ExpressionParser`1[T].ParseExpressionGeneric (System.String key, System.String expression, ContractConfigurator.ExpressionParser.DataNode dataNode) [0x00000] in <filename unknown>:0 
  at ContractConfigurator.Behaviour.Expression.OnLoad (.ConfigNode configNode) [0x00000] in <filename unknown>:0 
  at ContractConfigurator.ContractBehaviour.Load (.ConfigNode configNode) [0x00000] in <filename unknown>:0 
  at ContractConfigurator.ContractBehaviour.LoadBehaviour (.ConfigNode configNode, ContractConfigurator.ConfiguredContract contract) [0x00000] in <filename unknown>:0 
  at ContractConfigurator.ConfiguredContract.OnLoad (.ConfigNode node) [0x00000] in <filename unknown>:0 
```

**Comment by jrossignol** [comment]
Do you still have the save that causes this issue?  Can you post it here?

**Comment by pson** [comment]
I am sorry, I had played for two hours when that happened, and I played for two hours more so I doubt that any save I have will be helpful. I should have thought to exit the game and save the save.

**Comment by jrossignol** [comment]
That's alright - taking a closer look at this one, I think it was something that got fixed in 1.15.3.  Once you update you shouldn't run into this one again (looks like it wasn't all that frequent at least).
